pyseg/protocols/protocol_2d_classification.py

```diff
@@ -283,8 +283,22 @@ def _methods(self):
         return summary
 
     def _validate(self):
-        nSubtomos = self.inputSubtomos.get().getSize()
         errors = []
+        tol = 1e-3
+        inMask = self.inMask.get()
```

Comment:
All these will fail if not populated, right?

Reply:
@pconesa , when you say 'not populated' you mean if the mask is not provided? The program needs it, so it will be always present (allowsNull=False)